web/views/page_view.ex

```diff
@@ -1,10 +1,16 @@
 defmodule Nermesterts.PageView do
   use Nermesterts.Web, :view
 
-  def name_of_the(game) do
-    case game do
-      %{name: name} -> "<h3>" <> name <> "</h3>"
-      _ -> ""
-    end
+  def selected_game_phrase(%{message: message, has_token: true}, %{name: name}) do
+    String.replace(message, "#GAME#", "<h3>" <> name <> "</h3>")
   end
+  def selected_game_phrase(%{message: _, has_token: true}, _) do
+    ""
+  end
+  def selected_game_phrase(phrase, _) do
+    selected_game_phrase(phrase)
+  end
+
+  def selected_game_phrase(%{message: message, has_token: false}), do: message
```

Comment:
@jonahrr @ericworkman 

Could we add the type to the Structs in the file just to make sure we're always passing the correct data around?